src/strategy/periodic.rs

```diff
@@ -2,19 +2,35 @@
 
 use crate::config::inputs;
 use crate::weekly::{utils, WeeklyCalendar, WeeklyWindow};
+use chrono::{TimeZone, Utc};
 use failure::{Error, Fallible};
 use futures::future;
 use futures::prelude::*;
 use log::trace;
 use serde::Serialize;
+use std::fs::read_link;
+use std::path::Path;
 use std::pin::Pin;
 use std::time::Duration;
+use tzfile::Tz;
 
 /// Strategy for periodic (weekly) updates.
-#[derive(Clone, Debug, Default, Serialize)]
+#[derive(Clone, Debug, Serialize)]
 pub(crate) struct StrategyPeriodic {
     /// Whitelisted time windows during which updates are allowed.
     schedule: WeeklyCalendar,
+    /// Time zone in which time windows are defined in.
+    #[serde(skip_serializing)]
```

Comment:
`tzfile::Tz` currently does not implement `Serialize`.

Reply:
From an out-of-band chat: we should consider adding `serde` support directly to that crate, possibly behind a feature gate.